src/platform_impl/macos/util/mod.rs

```diff
@@ -131,16 +131,13 @@ pub unsafe fn ns_string_to_rust(ns_string: id) -> String {
   string.to_owned()
 }
 
-#[allow(dead_code)] // In case we want to use this function in the future
-pub unsafe fn app_name() -> Option<id> {
-  let bundle: id = msg_send![class!(NSBundle), mainBundle];
-  let dict: id = msg_send![bundle, infoDictionary];
-  let key = ns_string_id_ref("CFBundleName");
-  let app_name: id = msg_send![dict, objectForKey:*key];
-  if app_name != nil {
-    Some(app_name)
-  } else {
-    None
+/// Gets the app's name from the `localizedName` property of `NSRunningApplication`
+pub fn app_name() -> String {
+  unsafe {
+    let app_class = class!(NSRunningApplication);
+    let app: id = msg_send![app_class, currentApplication];
+    let name: id = msg_send![app, localizedName];
+    ns_string_to_rust(name)
```

Comment:
just took a glance at the relevant docs and seems like `localizedName`  could be undefned and also could be a localized name. I don't use mac so you'd have to cnofirm whether native Quit men item changes the app name to a localized version or keeps the same no matter what system language is used.

Also this function should return an Option like it used to be

you can also keep the current implementatio  as is and add a new function that calls this and converts the id to a string cause I think it might be need to have access to the pointer at some point.

Reply:
> just took a glance at the relevant docs and seems like localizedName could be undefned

Oh, my bad, I'll look into that. I didn't think it could be undefined but I'll check it out.

> I don't use mac so you'd have to cnofirm whether native Quit men item changes the app name to a localized version or keeps the same no matter what system language is used.

Normal native behaviour is to display the localized name.

> Also this function should return an Option like it used to be

I changed it because the current implementation always returns a value, it can never be `None`, and I was changing the signature anyway… but if you're right that `localizedName` could be undefined it will indeed need to be an `Option`.

> you can also keep the current implementatio as is and add a new function that calls this and converts the id to a string cause I think it might be need to have access to the pointer at some point.

Ok, I can do that if you prefer.